packages/nextjs/app/builders/0xAA87E8feDc5B7aCc46eE5BB12Af8E866e7C0c306/page.tsx

```diff
@@ -0,0 +1,44 @@
+import Image from "next/image";
+import Link from "next/link";
+import { Address } from "../../../components/scaffold-eth/Address";
```

Comment:
Nitpick: this can be shortened using ~~/

![image](https://github.com/BuidlGuidl/batch2.buidlguidl.com/assets/2486142/deecfc6a-f94f-4c7f-b36a-d91b58d1f97c)

Reply:
done